modules/steam.nix

```diff
@@ -409,5 +442,51 @@ in
         WINE_CPU_TOPOLOGY = "8:0,1,2,3,4,5,6,7";
       };
     })
+    (mkIf cfg.autoStart {
+      assertions = [
+        {
+          assertion = !config.systemd.services.display-manager.enable;
+          message = "No Display Managers must be enabled when jovian.steam.autoStart is used";
+        }
+      ];
+
+      services.xserver = {
+        enable = true;
+        displayManager.lightdm.enable = false;
+        displayManager.startx.enable = true;
+      };
+
+      services.greetd = {
+        enable = true;
+        settings = {
+          default_session = {
+            user = cfg.user;
+            command = "${pkgs.jovian-greeter}/bin/jovian-greeter";
+            service = "jovian-greeter";
+          };
+        };
+      };
+
+      security.pam.services = {
+        # Use a separate service to not result in a logind session for
+        # the greeter itself
+        jovian-greeter = {};
+
+        greetd.text = ''
```

Comment:
/me screams in PAM

Reply:
(Resolving as nothing has to be actioned here.)